bin/assigner/src/main.cpp

```diff
@@ -632,6 +640,26 @@ int main(int argc, char *argv[]) {
         return 1;
     }
 
+
+    std::map<std::string, nil::blueprint::print_format> print_circuit_output_options{
+        {"dec", dec},
+        {"hex", hex},
+        // {"print_circuit_output_bin", bin},
```

Comment:
Do we really need this? If not, please delete the option. If yes, please implement it.